nakadi-scopes.txt

```diff
@@ -0,0 +1,7 @@
+Resource Type ID: nakadi_event
+Resource Type Name: Nakadi Event
+Owning team: aruha
+Application Scope: nakadi_event.read
```

Comment:
I think we need a scope to create/update/list event types as of https://techjira.zalando.net/browse/ARUHA-50 right?